modules/ipi-install-preparing-the-bare-metal-node.adoc

```diff
@@ -81,6 +81,11 @@ Replace `<num>` for the worker number of the bare metal node in the two `name` f
 +
 Refer to the BMC addressing section for additional BMC configuration options. Replace `<protocol>` with the BMC protocol, such as IPMI, RedFish, or others.
 Replace `<bmc-ip>` with the IP address of the bare metal node's baseboard management controller.
++
+[NOTE]
+====
+If the MAC address of an existing bare metal node matches the MAC address of a bare metal host that you are attempting to provision, then the Ironic installation will fail. If the host enrollment, inspection, cleaning, or other Ironic steps fail, the `metal3-baremetal-operator` will continuously retry. See xref:modules/ipi-install-diagnosing-duplicate-mac-address.adoc#ipi-install-diagnosing-duplicate-mac-address_{context}[Diagnosing a host duplicate MAC address] for more information.
```

Comment:
While I find "bare metal node" and "bare metal host" perfectly clear in this note, I was just schooled in using our supplementary style guide.

Maybe the same "...the `metal3-baremetal-operator` retries the installation continuously." change.

Reply:
Yes, I've made this change using proper name - Bare Metal Operator